.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -8,12 +8,15 @@ labels: ''
 assignees: ''
 
 ---
+I have read check documentation: https://checkstyle.org/config_xxxxxx.html#NameOfAffectedCheck
 
-Check documentation: https://checkstyle.org/config_xxxxxx.html#NameOfAffectedCheck
+I have downloaded the latest checkstyle from: https://checkstyle.org/cmdline.html#Download_and_Run
+
```

Comment:
remove this empty line.

Reply:
After removing empty lines the text is rendered somewhat like: 
![image](https://user-images.githubusercontent.com/48255244/104820518-29b61880-585b-11eb-8904-fda883c6f441.png)

Reply:
Adding empty line takes these three lines on their own seperate line.